src/one_dragon/base/controller/pc_screenshot/gdi_screencapper_base.py

```
@@ -0,0 +1,350 @@
+            return False
+
+    def ensure_bitmap_resources(self, width, height, hwndDC_for_create=None):
+        needs_create = (self.saveBitMap is None or self.width != width or self.height != height)
+        if needs_create:
+            with self._lock:
+                if (self.saveBitMap is None or self.width != width or self.height != height):
+                    if self.saveBitMap:
+                        if self.mfcDC and getattr(self, "_selected_prev", None) is not None:
+                            try:
+                                ctypes.windll.gdi32.SelectObject(self.mfcDC, self._selected_prev)
+                            except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)